src/util.ts

```diff
@@ -35,6 +36,15 @@ function checkIsMinGW(): boolean {
   }
 }
 
+export async function resolveGlobs(globs: string[]) {
+  let paths: string[] = [];
+  for (const glob of globs) {
```

Comment:
Had trouble getting a reduce to work for Array.

Reply:
Isn't `map` more appropriate here than `reduce`?

```javascript
return globs.map(async (glob) => await globby(glob, {gitignore: true} as any));
```

Reply:
Wouldn't that return an Array of Arrays?

Reply:
Oops, you're right. Actually, I don't think either is needed, as `globby` takes a string array as the first argument, so you could do:

```javascript
return await globby(globs, {gitignore: true} as any);
```

Reply:
Oh, you are correct 😅 will update when I get to my computer.